Upstream-Status: Backport [https://gitlab.xfce.org/thunar-plugins/thunar-shares-plugin/-/commit/e4bce21a]

Signed-off-by: Kai Kang <kai.kang@windriver.com>

From e4bce21a1bd744ccfde84de4669ac30aa1d72135 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ga=C3=ABl=20Bonithon?= <gael@xfce.org>
Date: Fri, 2 Dec 2022 20:18:46 +0100
Subject: [PATCH] build: Bump GLib minimum required to 2.26

This is the minimum required to build since xfce4-dev-tools >= 4.17.1
set GLIB_VERSION_MAX_ALLOWED to the GLib version found in
configure.ac.in.

MR: !3
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index a458785..bc6d24c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -61,7 +61,7 @@ dnl -------------------------------------------------------
 dnl Packages checks
 dnl -------------------------------------------------------
 XDT_CHECK_PACKAGE([GTK], [gtk+-3.0], [3.22.0])
-XDT_CHECK_PACKAGE([GLIB], [glib-2.0], [2.18.0])
+XDT_CHECK_PACKAGE([GLIB], [glib-2.0], [2.26.0])
 XDT_CHECK_PACKAGE([THUNARX], [thunarx-3], [1.0.1])
 
 dnl -------------------------------------------------------
-- 
2.34.1

